config/rbac/kustomization.yaml

```diff
@@ -12,7 +12,7 @@ resources:
 # Comment the following 4 lines if you want to disable
 # the auth proxy (https://github.com/brancz/kube-rbac-proxy)
 # which protects your /metrics endpoint.
-- auth_proxy_service.yaml
-- auth_proxy_role.yaml
-- auth_proxy_role_binding.yaml
-- auth_proxy_client_clusterrole.yaml
+# - auth_proxy_service.yaml
```

Comment:
so we do not want to "protect" our metrics endpoint? I assume thats a bit of a misnomer since there should be lots of other protections in place preventing this from exposing the metrics endpoint, but wanted to double check.

Reply:
yes, if you have access to the cluster you can read metrics from the service without having a specific role assigned. I'm not aware of any service that uses the rbac proxy in OSD, but of many who don't, so I think it's consistent with other operators.